app/bot.go

```diff
@@ -143,3 +144,37 @@ func (a *App) UpdateBotOwner(botUserId, newOwnerId string) (*model.Bot, *model.A
 
 	return result.Data.(*model.Bot), nil
 }
+
+// disableUserBots disables all bots owned by the user given
+func (a *App) disableUserBots(userId string) *model.AppError {
```

Comment:
Should we just expose a SQL store method to do this efficiently/without for loops?